lib/Conversion/TritonGPUToLLVM/Utility.cpp

```diff
@@ -167,16 +167,16 @@ Value storeShared(ConversionPatternRewriter &rewriter, Location loc, Value ptr,
 
 static Value commonShflSync(Location loc, ConversionPatternRewriter &rewriter,
                             Value val, int i, const std::string &shuffleType,
-                            const std::string &clamp) {
+                            const std::string &clamp, Value laneId) {
```

Comment:
`laneId` is not used for NV path, what do you think about adding default value for laneId?
```suggestion
                            const std::string &clamp, Value laneId = Value()) {
```

This way you reduce divergence from upstream by 3 lines.
Two in this function and one in
```
Value shflSync(Location loc, ConversionPatternRewriter &rewriter, Value val,
               int i) {
  return commonShflSync(loc, rewriter, val, i, "bfly", "0x1f");   <-- THIS LINE
}
```